extmod/moducryptolib.c

```diff
@@ -234,19 +310,35 @@ STATIC mp_obj_t aes_process(size_t n_args, const mp_obj_t *args, bool encrypt) {
         }
     }
 
-    if (self->block_mode == UCRYPTOLIB_MODE_ECB) {
+    switch (self->block_mode) {
+    case UCRYPTOLIB_MODE_ECB: ;
```

Comment:
MicroPython codebase uses proper block scope in such cases.

Reply:
Done